ingenialink/canopen/dictionary.py

```diff
@@ -14,35 +18,42 @@ class CanopenDictionary(Dictionary):
 
     """
 
-    class AttrRegCanDict(Dictionary.AttrRegDict):
-        IDX = "idx"
-        SUBIDX = "subidx"
-
-    def __init__(self, dictionary_path):
+    def __init__(self, dictionary_path: str) -> None:
```

Comment:
Remove type from docstring

Reply:
This comment applies to all functions that you changed.

Reply:
Done!